data/NostrFlu.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Nostr Flu
+web: https://scrapbox-reader.vercel.app/nostr/NostrFlu
+git: https://github.com/heguro/nostr-following-list-util
+instances:
+tags:
+- tool
+platforms:
```

Comment:
I now see this link: https://heguro.github.io/nostr-following-list-util/

Platform looks like web, not some OS.

Reply:
Okay. Perhaps a nostr.info methodology is needed? 

I have currently have this: 

![image](https://user-images.githubusercontent.com/44260360/228105515-d881f167-9efb-44bc-be09-f509d97b30a6.png)